APIAssignment/src/test/java/APIAssignments/ConfigReader.java

```diff
@@ -0,0 +1,36 @@
+package APIAssignments;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigReader {
+    public static Properties getProperties() throws IOException {
+        String userDirectory = System.getProperty("user.dir");
+        FileInputStream file = new FileInputStream(userDirectory + "/config.properties");
+        Properties properties = new Properties();
+        properties.load(file);
+        return properties;
+    }
+
+    public static String getUrl() throws IOException {
```

Comment:
No need to write the ConfigReader code in this format....you have hard coded the values

Reply:
Done !